server/coderadar-webapp/src/main/java/org/wickedsource/coderadar/security/registration/rest/RegistrationController.java

```diff
@@ -0,0 +1,33 @@
+package org.wickedsource.coderadar.security.registration.rest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.wickedsource.coderadar.security.domain.UserCredentialsResource;
+
+@Controller
+@Transactional
+@RequestMapping(path = "/user/register")
+public class RegistrationController {
+
+    private RegistrationService registrationService;
+
+    @Autowired
+    public RegistrationController(RegistrationService registrationService) {
+        this.registrationService = registrationService;
+    }
+
+    @RequestMapping(method = RequestMethod.POST)
+    public ResponseEntity register(@RequestBody UserCredentialsResource userCredentialsResource) {
```

Comment:
The register function should take a "UserRegistrationDataResource" as input (since in future, there will be more properties that username and password) and return a UserResource with a "self" link so that the client knows the ID of the user just registered. The UserResource may contain just the username for now.